source/getting-started/onboarding.markdown

```diff
@@ -3,56 +3,55 @@ title: "Onboarding Home Assistant"
-   - Select **Create account**.
+This will create an administrator account. It will always be able to change everything in Home Assistant. Enter your name, username, and password and select **Create Account**.   Be sure to save this information safely as there is no way to recover it.
 
-    ![Set your username and password.](/images/getting-started/username.png)
+![Set your username and password.](/images/getting-started/username.png)
 
-4. Enter the location of your home.
-   - The location is used to populate settings such as time zone, unit system, and currency.
-   - It is also used for location-based information and automations: for example showing the weather-forecast, opening the shades at sunrise, or starting the vacuum when you leave the home.
-   - If you'd rather not send your location, you can choose a location far away from where you live.
-   - You can always change this information later in the settings.
+If you want to restore from a backup of a previous installation, select **Restore from backup**.
 
-    ![Define your location.](/images/getting-started/onboarding_location.png)
+Continue with the procedure on restoring from backup. You can ignore the remaining steps listed on this page
 
-5. Select which information you are willing to share.
-    - Sharing is disabled by default. However, we would like to encourage you to share some of this data.
-    - This information helps us to find out which platforms we need to support and where to focus our efforts.
-    - The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/).
-   ![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png)
+### Step 3
 
-6. Once you are done, select **Next**.
-    - Home Assistant will then show any {% term devices %} it has discovered on your network.
-    - Don't be alarmed if you see fewer items than shown below; you can always manually add devices later.
+Enter the location of your home. The location is used to configure the time zone, unit system, and currency. It is also used for automations that would show the weather, open the shades at sunrise, or start the vacuum when you leave the home. You can always change this information later in settings.
 
-    ![Discovery of devices on your network.](/images/getting-started/onboarding_devices.png)
+![Define your location.](/images/getting-started/onboarding_location.png)
 
-7. Finally, select **Finish**.
-   - Now you're brought to the Home Assistant web interface. If some of your devices were discovered and set up automatically, this default dashboard may already show some of your devices.
+ After finding your location, select **Next**
 
-{% include getting-started/next_step.html step="Concepts & Terminology" link="/getting-started/concepts-terminology/" %}
+### Step 4
+
```

Comment:
Was this step deleted on purpose? 
*Select which information you are willing to share.*
We probably still want them to select these options.